src/python/WMCore/MicroService/MSRuleCleaner/MSRuleCleaner.py

```diff
@@ -404,6 +418,47 @@ def setPlineMarker(self, wflow, pName):
                 wflow['CleanupStatus'][pName] = False
         return wflow
 
+    def _getStatusTransitionTime(self, wflow, status=None):
```

Comment:
Given that you pass the wflow object, I'd say the `status` parameter is irrelevant here.

Reply:
Agreed! This would simplify things. Working on that  now.